fem/conduitdatacollection.hpp

```diff
@@ -209,6 +209,9 @@ class ConduitDataCollection : public DataCollection
                                                      const conduit::Node &n_field,
                                                      bool zero_copy = false);
 
+   /** Checks if any rank in the mesh has boundary elements */
+   static bool HasBoundaryElements(mfem::Mesh *mesh);
```

Comment:
[The following comment is only relevant if `HasBoundaryElements()` does not move to `mfem::Mesh` / `mfem::ParMesh`]

The `HasBoundaryElements` function was useful for `SidreDataCollection`. Since it's being used in `ConduitDataCollection` as well, it seems like it should be upgraded to the `DataCollection` base class.